build-system/server/app-index/file-list.js

```diff
@@ -72,12 +72,8 @@ const linksToExample = (shouldContainBasepath, opt_name) =>
   examplesPathRegex.test(shouldContainBasepath) &&
   htmlDocRegex.test(opt_name || shouldContainBasepath);
 
-const ExamplesSelectModeOptional = ({basepath, selectModePrefix}) =>
-  !examplesPathRegex.test(basepath + '/')
-    ? ''
-    : ExamplesDocumentModeSelect({
-        selectModePrefix,
-      });
+const ExamplesSelectModeOptional = ({basepath}) =>
+  !examplesPathRegex.test(basepath + '/') ? '' : ExamplesDocumentModeSelect();
```

Comment:
This is a nice fix. The files in `build-system/server` are a whole other beast, so it's good to see them being checked.